### DISCUSSION TITLE
How can I enable draggable functionality for the Y-axis in finplot?

### DISCUSSION THREAD

**Discussion by NijanJ** [highfestiva/finplot#423] [Q&A]
I'm encountering an issue with finplot where the Y-axis is not draggable. I would like to enable the ability to drag the Y-axis in order to adjust the scale manually. Could you please provide guidance on how to achieve this?

Thank you!

**Comment by highfestiva** [comment]
Current implementation is `Ctrl+Mouse ScrollWheel`. If you want something better, you'll have to do a bit of coding.
